### PR TITLE
update rails to 5.1.0.rc1

### PR DESCRIPTION
**Pull request by pavel-zhuravlyov**

### CODE REVIEW

File: config/cable.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+development:
```
Comment (rilian) [inline config/cable.yml]: let's remove this

File: config/secrets.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+# Shared secrets are available across all environments.
+
+shared:
+  api_key: 123
+
+# Environmental secrets are only available for that specific environment.
+
+development:
+  secret_key_base: fee2c21976553d5d0198a586e8db23425033b36673abfb176f4356046b3effff721879f14bb7df756a8f487fca476bd991e004bbcdc18dc30f5c9d1cdd12c25d
```
Comment (rilian) [inline config/secrets.yml]: replace to ENV value plz

**Comment by rilian** [comment]
plz upd to rc2

**Comment by rilian** [comment]
plz remove cable and spring

**Comment by pavel-zhuravlyov** [comment]
plz review

**Comment by rilian** [comment]
ok but i will test before merging

**Pull request closed without merging**